### PR TITLE
add support for native_ipv6 in vrrp instances

### LINKED ISSUE #183: Please add support for "native_ipv6" vrrp_instance statement
Please add support for "native_ipv6" vrrp_instance statement to allow a instance to be forced to use ipv6. A ipv6 vrrp_instance without the "native_ipv6" keyword does not configure the virtual ipv6 address with the "deprecated nodad" options.

### PR DESCRIPTION
**Pull request by sigbjorntux**
This pull request Fixes #183

### CODE REVIEW

File: manifests/vrrp/instance.pp [hunk 1]
```diff
@@ -176,6 +179,7 @@
   $dont_track_primary                       = false,
   $use_vmac                                 = false,
   $vmac_xmit_base                           = true,
+  $native_ipv6                              = undef,
```
Comment (bastelfreak) [inline manifests/vrrp/instance.pp]: Should this be a boolean?
```suggestion
  Boolean $native_ipv6                      = false,
```

**Comment by bastelfreak** [comment]
Hi @sigbjorntux, thanks for the PR, can you please add a test for it and document it in the README.md?

**Comment by sigbjorntux** [comment]
> Hi @sigbjorntux, thanks for the PR, can you please add a test for it and document it in the README.md?

Hi @bastelfreak. If you want me to update the README.md I assume you are looking for an example with a mixed ipv4/ipv6 instance configuration in a sync group, in a new section? 

For the test I would need some guidance. Please let me know if my last commit 2780bf9 would be sufficient.

**Comment by bastelfreak** [comment]
Hi. the test looks fine. It's helpful when a parameter is documented in the readme. A working example is even better.

**Comment by sigbjorntux** [comment]
> Hi. the test looks fine. It's helpful when a parameter is documented in the readme. A working example is even better.

Ok. I will look into a README example.

Travis does not seem to agree with the test being ok. Any idea what might be wrong?

File: templates/vrrp_instance.erb [hunk 2]
```diff
@@ -31,6 +31,9 @@ vrrp_instance <%= @_name %> {
   <%- if @vmac_xmit_base -%>
   vmac_xmit_base
   <%- end -%>
+  <%- if @native_ipv6 -%>
```
Comment (bastelfreak) [inline templates/vrrp_instance.erb]: take a look at some lines above. Your codeblock is within the `use_vmac` condition. I am not sure if this is on purpose or not (I assume not).

Comment (sigbjorntux) [reply templates/vrrp_instance.erb]: Ah, right. An excellent example of why tests are a good thing to have. :) Should be fixed in 5750127.

**Comment by sigbjorntux** [comment]
Adding the config option global_tracking to sync_groups. This allows health checks for vrrp instances to continue working when they are added to a sync_group, which is needed when having VI for both ipv4 and ipv6 addresses.

**Comment by sigbjorntux** [comment]
Hi @bastelfreak, a documentation example has been provided. Please review and merge. Thanks.

**Comment by bastelfreak** [comment]
Thanks for the work!

**Pull request merged**